app/operations/kinesis/count_classification.rb

```diff
@@ -29,18 +29,20 @@ def execute
           StudentUser.increment_counter :classifications_count, student_user_id
         end
 
-        StudentAssignment.joins(:assignment).where(assignments: {workflow_id: workflow_id},
-                                                   student_user_id: student_user_ids)
-                                            .pluck(:id).each do |student_assignment_id|
-          StudentAssignment.increment_counter :classifications_count, student_assignment_id
+        sas = StudentAssignment
+              .joins(:assignment)
+              .where(assignments: {workflow_id: workflow_id}, student_user_id: student_user_ids).each do |sa|
```

Comment:
Avoid abbreviated model names like `sa`, prefer `student_assignment`.

Reply:
I think it's arguable in a block this short when the full name would force an inner line to wrap, but okay.